build/patches/Partition-DNS-requests-by-top-frame-NIK.patch

```diff
@@ -342,17 +342,17 @@ diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
        : session_(session),
          hostname_(std::move(hostname)),
          qtype_(qtype),
-@@ -1122,7 +1132,8 @@ class DnsTransactionImpl : public DnsTransaction,
-         attempts_count_(0),
-         had_tcp_retry_(false),
-         resolve_context_(resolve_context->AsSafeRef()),
--        request_priority_(DEFAULT_PRIORITY) {
-+        request_priority_(DEFAULT_PRIORITY),
-+        network_isolation_key_(network_isolation_key) {
+@@ -1115,7 +1125,8 @@ class DnsTransactionImpl : public DnsTransaction,
+         secure_dns_mode_(secure_dns_mode),
+         fast_timeout_(fast_timeout),
+         net_log_(net_log),
+-        resolve_context_(resolve_context->AsSafeRef()) {
++        resolve_context_(resolve_context->AsSafeRef(),
++        network_isolation_key_(network_isolation_key)) {
```

Comment:
the error is here, it is the parenthesis in the wrong position, is:

```
        resolve_context_(resolve_context->AsSafeRef()),    <--- DOUBLE ")"
        network_isolation_key_(network_isolation_key) {    <--- SINGLE ")"
```

Reply:
Thanks, I had skipped this patch but I will add it back now.